File: db-validation/python/db_test.py
import psycopg2

def test_order_status():
    conn = psycopg2.connect(
        dbname="testdb",
        user="testuser",
        password="password",
        host="localhost"
    )
    cur = conn.cursor()
    cur.execute("SELECT status FROM orders WHERE order_id = 101;")
    status = cur.fetchone()[0]

    assert status == "COMPLETED"

    cur.close()
    conn.close()

File: db-validation/java/db_test.java
import org.junit.Test;
import java.sql.Connection;
import java.sql.DriverManager;
import java.sql.ResultSet;
import java.sql.Statement;
import static org.junit.Assert.assertEquals;

public class DBTest {

    @Test
    public void testOrderStatus() throws Exception {
        Connection conn = DriverManager.getConnection(
                "jdbc:postgresql://localhost:5432/testdb",
                "testuser",
                "password"
        );

        Statement stmt = conn.createStatement();
        ResultSet rs = stmt.executeQuery("SELECT status FROM orders WHERE order_id = 101;");

        if (rs.next()) {
            String status = rs.getString("status");
            assertEquals("COMPLETED", status);
        }

        rs.close();
        stmt.close();
        conn.close();
    }
}
